Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs

```diff
@@ -0,0 +1,253 @@
+            try
+            {
+                SyncSolution();
+                switch (EditorUserBuildSettings.activeBuildTarget)
+                {
+                    case BuildTarget.WSAPlayer:
+                        success = await UwpPlayerBuildTools.BuildPlayer(new UwpBuildInfo(true) { BuildAppx = true });
+                        break;
+                    default:
+                        var buildInfo = new BuildInfo(true) as IBuildInfo;
+                        ParseBuildCommandLine(ref buildInfo);
+                        var buildResult = BuildUnityPlayer(buildInfo);
+                        success = buildResult.summary.result == BuildResult.Succeeded;
+                        break;
+                }
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"Build Failed!\n{e.Message}\n{e.StackTrace}");
+                success = false;
+            }
+
+            Debug.Log($"Exiting command line build... Build success? {success}");
+            EditorApplication.Exit(success ? 0 : 1);
+        }
+
+        internal static bool CheckBuildScenes()
+        {
+            if (EditorBuildSettings.scenes.Length == 0)
+            {
+                return EditorUtility.DisplayDialog("Attention!",
+                                                   "No scenes are present in the build settings.\n" +
+                                                   "The current scene will be the one built.\n\n" +
+                                                   "Do you want to cancel and add one?",
+                                                   "Continue Anyway", "Cancel Build");
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Get the Unity Project Root Path.
+        /// </summary>
+        /// <returns>The full path to the project's root.</returns>
+        public static string GetProjectPath()
+        {
+            return Path.GetDirectoryName(Path.GetFullPath(Application.dataPath));
+        }
+
+        public static void ParseBuildCommandLine(ref IBuildInfo buildInfo)
+        {
+            string[] arguments = Environment.GetCommandLineArgs();
+
+            for (int i = 0; i < arguments.Length; ++i)
+            {
+                switch (arguments[i])
+                {
+                    case "-autoIncrement":
+                        buildInfo.AutoIncrement = true;
+                        break;
+                    case "-scenes":
+                        // TODO parse json scene list and set them.
+                        break;
+                    case "-buildOutput":
+                        buildInfo.OutputDirectory = arguments[++i];
+                        break;
+                    case "-colorSpace":
+                        buildInfo.ColorSpace = (ColorSpace)Enum.Parse(typeof(ColorSpace), arguments[++i]);
+                        break;
+                    case "-x86":
+                    case "-x64":
```

Comment:
There's no command line switch for ARM. This may be needed on CI. I'm not sure whether we should add this here or to `UwpPlayerBuildTools.ParseBuildCommandLine`.

Reply:
I'll be sure to add :)